packages/office-ui-fabric-react/src/components/Rating/Rating.tsx

```diff
@@ -70,24 +70,38 @@ export class Rating extends BaseComponent<IRatingProps, IRatingState> {
   }
 
   public render() {
+    let id = this._id;
     let stars = [];
-
-    for (let i = this.props.min as number; i <= (this.props.max as number); i++) {
+    let starIds = [];
+    let {
+      disabled,
+      getAriaLabel,
+      max,
+      min,
+      rating,
+      readOnly,
+      size
+    } = this.props;
+
+    for (let i = min as number; i <= (max as number); i++) {
       let ratingStarProps: IRatingStarProps = {
         fillPercentage: this._getFillingPercentage(i),
-        disabled: this.props.disabled ? true : false
+        disabled: disabled ? true : false
       };
 
+      starIds.push(this._getStarId(i - 1));
+
       stars.push(
         <button
           className={ css('ms-Rating-button', styles.ratingButton, {
-            ['ms-Rating--large ' + styles.rootIsLarge]: this.props.size === RatingSize.Large,
-            ['ms-Rating--small ' + styles.rootIsSmall]: this.props.size !== RatingSize.Large
+            ['ms-Rating--large ' + styles.rootIsLarge]: size === RatingSize.Large,
+            ['ms-Rating--small ' + styles.rootIsSmall]: size !== RatingSize.Large
```

Comment:
size === RatingSize.Small?

Reply:
you can have a default value for size, but it seems error prone to consider everything not large as small.

Reply:
Just noticed this while i was here. (I know you didn't make it this way :)